### PR TITLE
Fix: fetch values automatically

### PR DESCRIPTION
**Pull request by IgnasKia**
After submit new evaluation all students fetches automatically without refreshing page.

### CODE REVIEW

File: src/app/views/home/home.component.ts [hunk 1]
```diff
@@ -16,42 +16,34 @@ import {Observable} from 'rxjs';
...
-  constructor(public dialog: MatDialog, private studentService: StudentService,  private mentorService: MentorService,
-              private evaluationService: EvaluationService) { }
+
+  constructor(public dialog: MatDialog, private studentService: StudentService, private mentorService: MentorService,
+              private evaluationService: EvaluationService) {
+  }
 
   ngOnInit(): void {
-    this.getMentorStudents();
+    this.students$ = this.studentService.getMentorStudents().pipe();
```
Comment (PovilasKM) [inline src/app/views/home/home.component.ts]: Why did you add `pipe()` here?

Comment (IgnasKia) [reply src/app/views/home/home.component.ts]: I did some testing with pipe() method and forgot to delete it. All good now

**Review by ignaspangonis — APPROVED** [review]
